src/ui/server/controllers/insurance/your-buyer/index.ts

```diff
@@ -0,0 +1,45 @@
+import { PAGES } from '../../../content-strings';
+import { Request, Response } from '../../../../types';
+import api from '../../../api';
+import { ROUTES, TEMPLATES } from '../../../constants';
+import insuranceCorePageVariables from '../../../helpers/page-variables/core/insurance';
+import { mapCountriesSelect } from '../../../helpers/mappings/map-countries-select';
+import yourBuyerDetailsValidation from './validation/yourBuyerDetailsValidation';
+import { yourBuyerFiledVariables } from '../../../content-strings/fields/insurance/your-buyer';
+
+const getMappedCountries = async (res: Response) => {
+  const countries = await api.external.getCountries();
+  if (!countries || !countries.length) {
+    return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
```

Comment:
Do you think we should have the `res.redirect` logic in the get/post itself? So we're not passing `res` around? Could return false in this function to flag if we need to redirect in the controller